Training2/src/task3/src/App.js

```diff
@@ -0,0 +1,74 @@
+import { useEffect, useState } from "react";
+import "./App.css";
+
+function App() {
+  const [value, setValue] = useState("");
+  const [valueCountDown, setValueCountDown] = useState(0);
+  const [isStop, setIsStop] = useState(false);
+  const handleChange = (e) => {
+    setValue(e.target.value);
+  };
+
+  useEffect(() => {
+      const countdown = setTimeout(() => {
+        if (valueCountDown>0 && isStop===false)
+        setValueCountDown(valueCountDown - 1);
+      }, 1000);
+      return ()=>clearTimeout(countdown);
```

Comment:
@phuongluongZigvy we only run clearTimeout whenever the setTimeout is running and we press stop, when you set clearTimeout like this, it's always run when valueCountDown changes, isStop false -> true, true -> false, and setTimeout created even isStop true -> false, you can try to use the setInterval for best solution